code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -689,15 +689,18 @@ GLOBAL_LIST_EMPTY(aide_list)
 /datum/reagent/flightpotion/reaction_mob(mob/living/M, method=TOUCH, reac_volume, show_message = 1)
 	if(iscarbon(M) && M.stat != DEAD)
 		var/mob/living/carbon/C = M
-		var/holycheck = ishumanbasic(C)
-		if(!(holycheck || islizard(C) || ismoth(C) || isskeleton(C) || ispreternis(C) || isipc(C) || (reac_volume < 5))) //humans (which are holy?), lizards, skeletons, and preterni(ises?) can get wings
+		var/valid_species = (ishumanbasic(C) || islizard(C) || ismoth(C) || isskeleton(C) || ispreternis(C) || isipc(C))
```

Comment:
ew this check sucks

Reply:
It's basically the original check but moved out of the if statement. I could rework it into more than one line of code and make it prettier but why would I bother when any performance gains would be pretty much nonexistant from how often this gets called.